public/app/features/dashboard/components/HelpWizard/utils.ts

```diff
@@ -304,3 +286,143 @@ const embeddedDataTemplate: any = {
   ],
   schemaVersion: 37,
 };
+
+export async function getDebugScene(panel: PanelModel, rand: Randomize, timeRange: TimeRange): Promise<SceneObject> {
+  const saveModel = panel.getSaveModel();
+
+  // reproducable
+  const data = await firstValueFrom(
+    panel.getQueryRunner().getData({
+      withFieldConfig: false,
+      withTransforms: false,
+    })
+  );
+
+  const dsref = panel.datasource;
+  const frames = randomizeData(getPanelDataFrames(data), rand);
+  const grafanaVersion = `${config.buildInfo.version} (${config.buildInfo.commit})`;
+  const queries = saveModel?.targets ?? [];
+
+  const scene = new SceneFlexLayout({
+    children: [
+      new SceneFlexLayout({
+        direction: 'column',
+        children: [
+          new VizPanel({
+            title: 'Reproduced with embedded data',
+            pluginId: saveModel.type,
+            options: saveModel.options,
+            fieldConfig: saveModel.fieldConfig,
+            $data: new SceneDataNode({
+              data: {
+                state: LoadingState.Done,
+                series: frames.map(dataFrameFromJSON),
+                timeRange,
+              },
+            }),
+          }),
+          new VizPanel({
+            title: 'Data from panel above',
+            pluginId: 'table',
+            size: { height: '30%' },
+            options: { showTypeIcons: true },
+            $data: new SceneDataNode({
+              data: {
+                state: LoadingState.Done,
+                series: frames.map(dataFrameFromJSON),
+                timeRange,
+              },
+            }),
+          }),
```

Comment:
this makes me want to make VizPanel generic, so options can be typed. Then have for core data sources have specific scene objects. the the "as code" story would be really nice. 

```ts
new TimeSeriesViz({ }) 
new TableViz({ }) 
```